src/main/java/core/basesyntax/service/operation/SupplyOperationHandler.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax.service.operation;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.StorageService;
+
+public class SupplyOperationHandler implements OperationHandler {
+    private final StorageService storageService;
+
+    public SupplyOperationHandler(StorageService storageService) {
+        this.storageService = storageService;
+    }
+
+    @Override
+    public void handle(FruitTransaction transaction, StorageService storageService) {
```

Comment:
The 'storageService' parameter shadows the field 'storageService'. Since you are already passing the 'StorageService' through the constructor, there is no need to pass it again as a method parameter. Please remove the parameter from the method signature to use the class field instead.